charts/ironic-standalone/README.md

```diff
@@ -0,0 +1,168 @@
+tftp:
+  externalIPs:
+    - 10.10.10.10
+mariadb:
+  db:
+    password: secret1
+  rootUser:
+    password: secret1
+  master:
+    persistence:
+      existingClaim: existing-claim-name
+rabbitmq:
+  rabbitmq:
+    password: secret3
+```
+(you don't have to define secrets, but this step is required for production, as random-generated secrets will change each helm roll-over and will not match persistent storage db)
+
+Another example of the config file, using ingress and dynamic pv provisioners (make sure the tftp storageClass used allows cross-pod mounting):
+```
+---
+ironicServerName: ironic.example.io
+api:
+  ingress:
+    enabled: true
+    hosts:
+      - ironic.example.io
+httpboot:
+  ingress:
+    enabled: true
+    hosts:
+      - ironicwww.example.io
+tftp:
+  externalIPs:
+    - 10.10.10.10
+  nodeSelector:
+    hostname: 10.10.10.10.xip.io
+  persistence:
+    storageClass: sharefs
+mysql:
+  mysqlPassword: secret1
+  mysqlRootPassword: secret2
+rabbitmq:
+  rabbitmq:
+    password: secret3
+```
+
+Then install a chart using command:
+```
+helm repo add ironic https://ironic.storage.googleapis.com
+helm install ironic/openstack-ironic-standalone -f my-site.yaml
+```
+
+Keystone Authentication (Optional)
+==================================
+
+This chart deploys only Openstack Ironic by default.
+You can enable Keystone service by providing extra configuration parameters in the deployment:
+
+```
+keystone:
+  enabled: true
+  admin_password: ADMIN-PASSWORD
+  ironic_user: ironic
+  ironic_password: IRONIC-USER-PASSWORD
+  ironic_project_name: service
+  ironic_service_name: ironic
+  externalIPs:
+    - 10.10.10.10
+```
+
+This configuration will enable Keystone authentication. With 2 pre-configured user accounts: `admin` and `ironic`.
+The `ironic` account can be used to perform baremetal operations.
+
+Configuration
+=============
+
+All configuration parameters are documented in the `values.yaml` file.
+
+Comparison to [Kolla](https://docs.openstack.org/ironic/latest/install/standalone.html)
+=======================================================================================
+
+Kolla provides a method to deploy a full Openstack suite. This chart installs only Openstack Ironic.
+The chart has a *minimalistic* approach.
+
+Why minimalistic?
+-----------------
+* Uses simple Docker images with no sophisticated entrypoints. Docker images are from Openstack rpms packaged by CentOS.
+* Depending services like MySQL, Rabbitmq are installed from production Helm charts
+
+Storage
+=======
+
+The chart can use an existing volume claim. It will create a claim, if there is no claim name provided.
+It uses `subPath` functionality to create sub directories on the existing claim.
+The existing claim name must be defined in two places: `persistentVolumeClaimName` and `mysql.persistence.existingClaim`.
+
+One can upload disk images using HTTP PUT request:
+
+```
+curl -T diskimage.qcow2 http://<httpboot_externalIP>
+```
+
+Note on tftpd
```

Comment:
Better use dnsmasq

Reply:
might be, but we actually never used PXE boot so hwe havent tested that